src/schemata/definitions/config/workflow-config.yml

```
@@ -16,7 +16,7 @@ properties:
           The duration by which the automatic navigation to the next step is
           delayed.
   contextInfoSettings:
-    $ref: '#/definitions/context-info/contextInfoSettings'
+    $ref: '#/definitions/context-info-settings'
```

```suggestion
    $ref: '#/definitions/context-info-schema/context-info-settings'
```
I don't have strong feelings on this, but grouping related schemas does make it a little nicer to work with imo.